File: POO/JS/UberVan.js
class UberVan extends Car{

    constructor(license, driver, typeCarAccepted, seatsMaterial) {    
        super(license, driver)
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    takePassenger(passenger){
        if (passenger == 6) {
            this.passenger = passenger;
        }else{
           console.log("Se necesita minimo 6 pasajeros"); 
        }
    }

    get givePassenger(){
        return this.passenger;
    }
}

File: POO/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car{
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver, 
    Map<String, Map<String, Integer>> typeCarAccepted, 
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }  
    
    public UberVan(String license, Account driver){
        super(license, driver);
    } 

    public void setPassenger(Integer passenger){
        if(passenger == 6){
            super.passenger = passenger;
        }else{
            System.out.println("UberVan necesita 6 pasajeros");
        }
    }
}